src/bessel.jl

```diff
@@ -499,7 +509,7 @@ Bessel function of the first kind of order 0, ``J_0(x)``.
 """
 function besselj0(x::BigFloat)
     z = BigFloat()
-    ccall((:mpfr_j0, :libmpfr), Int32, (Ptr{BigFloat}, Ptr{BigFloat}, Int32), &z, &x, ROUNDING_MODE[])
+    ccall((:mpfr_j0, :libmpfr), Int32, (Ref{BigFloat}, Ref{BigFloat}, Int32), z, x, ROUNDING_MODE[])
```

Comment:
I see...so when you pass an instance `z` of a mutable type for a `Ref` argument then it actually passes a pointer to the original data in `z`?